Formula/osi.rb

```diff
@@ -0,0 +1,38 @@
+class Osi < Formula
+  desc "Open Solver Interface"
+  homepage "https://github.com/coin-or/Osi"
+  url "https://github.com/coin-or/Osi/archive/releases/0.108.6.tar.gz"
+  sha256 "984a5886825e2da9bf44d8a665f4b92812f0700e451c12baf9883eaa2315fad5"
+  license "EPL-1.0"
+
+  depends_on "pkg-config" => :build
+  depends_on "coinutils"
```

Comment:
This should be test only

Reply:
Think it's needed everywhere. Without it:

```txt
checking pkg-config is at least version 0.16.0... yes
checking for COIN-OR package CoinUtils... not given: No package 'coinutils' found
configure: error: Required package CoinUtils not available.
configure: error: /bin/sh './configure' failed for Osi
```

Reply:
Okay, I only saw the reference in the test block.